### PR TITLE
MCR-2034 make maximal page scale configurable

### PR DESCRIPTION
**Pull request by sebhofmann**
the property is MCR.Viewer.Canvas.Startup.MaximalPageScale and default
is 4

[Link to jira](https://mycore.atlassian.net/browse/MCR-2034).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 8bc1c9b4603e346ce8332f05528471f4543c5a47! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/4402-pull_request-782

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 6788ebce310bfa096d47912d8a0a5953043d21ba! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/4406-pull_request-782

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 5d92da9a3b169250edf499d1e7e46dc563e03b2c! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/4414-pull_request-782

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 947b2297c36cd21dabe73a166faa4166a5fb0da0! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/4416-pull_request-782

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 8098ccc03b3e73fc7cf4ade66e97d027dca99761! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/4422-pull_request-782

File: mycore-viewer/src/main/java/org/mycore/viewer/configuration/MCRViewerBaseConfiguration.java [hunk 1]
```diff
@@ -102,6 +102,12 @@ public MCRViewerConfiguration setup(HttpServletRequest request) {
 
             addLocalCSS("default.css");
         }
+
+        String maximalScale = MCRConfiguration.instance().getString("MCR.Viewer.Canvas.Startup.MaximalPageScale", "");
```
Comment (kkrebs) [inline mycore-viewer/src/main/java/org/mycore/viewer/configuration/MCRViewerBaseConfiguration.java:106]: can you specify this property in mycore.properties also ?!

**Review by kkrebs — CHANGES_REQUESTED** [review]

**Review by kkrebs — APPROVED** [review]

**Pull request merged**